core/resharding_test.go

```diff
@@ -3,16 +3,47 @@ package core
 import (
 	"fmt"
 	"testing"
-)
 
-func TestFakeGetInitShardState(t *testing.T) {
-	ss := fakeGetInitShardState()
-	for i := range ss {
-		fmt.Printf("ShardID: %v, NodeList: %v\n", ss[i].ShardID, ss[i].NodeList)
-	}
-}
+	"github.com/harmony-one/harmony/core/types"
+	"github.com/stretchr/testify/assert"
+)
 
 func TestFakeNewNodeList(t *testing.T) {
 	nodeList := fakeNewNodeList(42)
 	fmt.Println("newNodeList: ", nodeList)
 }
+
+func TestShuffle(t *testing.T) {
+	nodeList := []types.NodeID{"node1", "node2", "node3", "node4", "node5", "node6", "node7", "node8", "node9", "node10"}
+	cpList := []types.NodeID{}
+	cpList = append(cpList, nodeList...)
+	Shuffle(nodeList)
+	cnt := 0
+	for i := 0; i < 10; i++ {
+		if cpList[i] == nodeList[i] {
+			cnt++
+		}
+	}
+	if cnt == 10 {
+		t.Error("Shuffle list is the same as original list")
```

Comment:
can you clarify what you are trying to test here

Reply:
When shuffle a list, we have a very low probability that the list stay the same

Reply:
maybe should do another type of checking